### PR TITLE
Add Winter 2017 events

### PR DESCRIPTION
**Pull request by magical**
Add the events we've done so far, a couple future events, and some
placeholders for the unscheduled weeks.

Week 2 description based on my email.

Week 3 description based on Connor's email.

It was really weird writing about the past events in future tense.
I just thought you should know that.

### CODE REVIEW

**Comment by magical** [comment]
@ConnorChristensen, @ElijahCaine Please take a look?

File: content/events/2017/winter2017/20170131_wk4.mkd [hunk 1]
```diff
@@ -0,0 +1,17 @@
+title: History of Hacking
+datetime: 2017-01-31 18:00:00
+category: events
+slug: meeting20160131
+preview: Erich Kramer (Halved) will take us on a journey through the history of hacking, from phreaking to botnets.
```
Comment (ElijahCaine) [inline content/events/2017/winter2017/20170131_wk4.mkd]: I would say `(Halved on freenode)`

Comment (magical) [inline content/events/2017/winter2017/20170131_wk4.mkd]: Done.

**Comment by magical** [comment]
@ElijahCaine Good point about adding tentative events; however, is it okay if i merge this now so we have *something* up? I'll add the rest this weekend.

**Comment by magical** [comment]
Merging.

**Pull request merged**